cfgov/prepaid_agreements/jinja2/prepaid_agreements/index.html

```diff
@@ -0,0 +1,100 @@
+        </div>
+
+        <div class="search_wrapper u-mt30">
+            <form class="u-mt15" method="get" action="." >
+                <div id="search-hidden-filters">
+                {% for filter in filters %}
+                    {% if filter in valid_filters %}
+                        {% for val in filters[filter] %}
+                            <input type="hidden" name="{{ filter }}" value="{{ val }}">
+                        {% endfor %}
+                    {% endif %}
+                {% endfor %}
+                </div>
+
+                <h3 class="h4 u-mb5">Search within</h3>
+                <div class="layout-row u-mb30">
+                    <div class="a-select flex-fixed">
+                        {% set opts = [
+                            ('All fields', 'all'),
+                            ('Product name', 'name'),
+                            ('Program manager', 'program_manager'),
+                            ('Other relevant party', 'other_relevant_parties')
+                        ]%}
+                        <select aria-label="Choose which field will be searched"
+                                id="search_field"
+                                name="search_field">
+                            <optgroup label="Search within">
+                                {% for label, val in opts %}
+                                <option value="{{ val }}"
+                                        {{ 'selected' if search_field == val else '' }}>
+                                    {{ label }}
+                                </option>
+                                {% endfor %}
+                            </optgroup>
+                        </select>
+                    </div>
+                    <div class="flex-all">
+                        <div class="m-btn-inside-input input-contains-label">
+                            <label for="searchText" class="input-contains-label_before input-contains-label_before__search">
+                                {{ svg_icon('search') }}
+                                <span class="u-visually-hidden">The term to search for</span>
+                            </label>
+                            <input id="searchText" 
+                                   type="text"
+                                   autocomplete="off"
+                                   class="a-text-input"
+                                   name="q"
+                                   placeholder="Enter search term"
+                                   value="{{ query }}">
+                            <a class="a-btn a-btn__link" href="{{ remove_url_parameter(request, { 'q': [query] }) }}">{{ svg_icon('remove') }} Clear</a>
+                        </div>
+                    </div>
+                    <button class="a-btn flex-fixed">Search</button>
+                </div>
+            </form>
+            <p class="u-mt30 u-mb15">
+                <a href="">View frequently used terms</a>
```

Comment:
What will this link do?

Reply:
This will link to a page with general info on searching the database / definitions of some of the metadata field terminology used on this page.